openapi.yaml

```diff
@@ -0,0 +1,1607 @@
+        - name: sample-summary-id
+          in: path
+          required: true
+          description: The UUID of the sample summary
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                collectionExerciseId:
+                  type: string
+                  format: uuid
+                  description: The UUID of the collection exercise
+      responses:
+        200:
+          description: The business's attributes have been updated
+        400:
+          description: The request body was missing collectionExerciseId
+  /businesses/search:
+    get:
+      tags:
+        - businesses
+      summary: Search for businesses
+      description: Search for businesses
+      parameters:
+        - name: query
+          in: query
+          required: true
+          description: The text to search for
+          schema:
+            type: string
+        - name: page
+          in: query
+          required: true
+          description: The page number of results
+          schema:
+            type: integer
+        - name: limit
+          in: query
+          required: true
+          description: The number of results per page
+          schema:
+            type: integer
+        - name: is_ru_ref_search
+          in: query
+          required: true
+          description: Whether the search is for an RU reference
+          schema:
+            type: boolean
+        - name: max_rec
+          in: query
+          required: true
+          description: The maximum number of records to search for in the database
+          schema:
+            type: integer
+      responses:
+        200:
+          description: The businesses have been retrieved
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  businesses:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/BusinessResponse'
+                  total_business_count:
+                    type: integer
+        400:
+          description: The request was missing one or more of the query parameters
+  /info:
+    get:
+      tags:
+        - info
+      summary: Get service information
+      description: Get service information
+      responses:
+        200:
+          description: The service information has been retrieved
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  name:
+                    type: string
+                    example: ras-party
+                  version:
+                    type: string
+                    example: 1.0.0
+  /parties:
+    post:
+      tags: 
+        - businesses
+      deprecated: true
```

Comment:
Missed why this is deprecated

Reply:
For all these I've put the new endpoint in the description. I was there when these were deprecated so going off my personal knowledge on this